apps/web/src/pages/templates/filter/FilterModal.tsx

```diff
@@ -196,3 +168,88 @@ const DeleteStepButton = styled(Button)`
   }
   margin-top: 0px;
 `;
+
+function getWebhookUrlGrid(control, stepIndex: number, index: number) {
```

Comment:
For this scenarios, I usuall prefer just doing a proper React component function instead of a function that returns a component. So it will look like: 

```
```suggestion
function WebHookUrlForm(control, stepIndex: number, index: number) {
```